### PR TITLE
TWEAK: Love pen now higly peaceful

### PR DESCRIPTION
**Pull request by ErdGinalD**
## What Does This PR Do

Replace Love in love pen with Pax, who give target full pacifism, instead of force status + reagent who just names "love" and doesnt do something to avoid you from death. Added reagent regeneration, so you can use it far more than 4 times for 20 TC.

## Why It's Good For The Game

Cheap stuffs always was the most funny or cursed ones, yet the pen only give you limited time for do something. Now with removing original author restrictions and adding regeneration, you can use it AND for easy kills without resistance AND for trolling sec who cant kill someone. Counter to this is pretty easy to obtain and dont restrict you from stunning opponents, so it will remain niche item. But now you can use it for fun, because it never deplete fully, and more fun = more joy to play. In my opinion, this pen now have all he need to be "niche fun item", without complaints.

## Images of changes

image

## Testing

Start local server, shot skrell, see how he cant hurt me, Jack. After this, i see how it regenerates inside with 100 second for every full shot.

<hr>

### Declaration

Pre-approve gang rises. After further conversation, original author agree 100 seconds is enough.
image

## Changelog

:cl:
tweak: Love pen now contain Pax instead of Love and slowly regenerate this reagent over time. In other words, he is usable and fun now.
/:cl:

### CODE REVIEW

File: code/modules/paperwork/pen.dm [hunk 1]
```diff
@@ -179,19 +179,25 @@
...
+	STOP_PROCESSING(SSobj, src)
+	return ..()
 
 /obj/item/pen/sleepy/love/fill_pen()
-	reagents.add_reagent("love", 100)
+	reagents.add_reagent("pax", 100) // strong and unique reagent, making you a pacifist for a long time.
+
+/obj/item/pen/sleepy/love/process()
+	if(reagents.total_volume < 100)
+		reagents.add_reagent("pax", 0.5) // slow refill over time. In average 1 dose every 100 seconds.
```
Comment (1080pCat) [inline code/modules/paperwork/pen.dm:199]: so my main worry with this, is that you dont need to wait the full 1.5 minutes to inject someone, you can just give alot of people small doses that last ~5 seconds if you're doing it every 0.5 units.

You may want to add a minimum transfer limit

**Comment by Crazyhair213** [comment]
Oh hey, I’m in this!! Also cool skrell item to antag with😁

File: code/datums/uplink_items/uplink_traitor.dm [hunk 2]
```diff
@@ -337,7 +337,7 @@
 //skrell
 /datum/uplink_item/species_restricted/lovepen
 	name = "Aggression Suppression Pen"
-	desc = "A syringe disguised as a functional pen which is filled with a potent aggression supressing chemical. The pen holds four doses of the mixture and it cannot be refilled."
+	desc = "A hypospay disguised as a functional pen which is filled with a potent aggression supressing chemical. The pen holds four doses of the mixture, slowly regenerates over time, but cannot be refilled."
```
Comment (lewcc) [inline code/datums/uplink_items/uplink_traitor.dm:340]:
```suggestion
	desc = "A hypospray disguised as a functional pen which is filled with a potent aggression suppressing chemical. The pen holds four doses of the mixture which slowly regenerates over time, but cannot be refilled."
```

**Review by lewcc — APPROVED** [review]

**Comment by Henri215** [comment]
@silverplatedelta You forgot the "requires wiki update" label, always verify if it needs one.

**Pull request merged**